go.sum

```diff
@@ -495,13 +495,6 @@ github.com/onosproject/onos-config v0.0.0-20191116095118-2ca4b5f98ae2/go.mod h1:
 github.com/onosproject/onos-config v0.0.0-20200124182344-6c3f5eb1e6ed/go.mod h1:rQfBju27qOGvL7Qtvxo6H9TlvU0oEw91emQzS14hmnE=
 github.com/onosproject/onos-config v0.0.0-20200131171925-566694cf7e94/go.mod h1:3OKwIzkqat9YiJ/DM1aRdBIWD5HANzbc5nfY8/eJAJM=
 github.com/onosproject/onos-config v0.0.0-20200204191831-5c2803ee469d/go.mod h1:rVNKd9v/Y4ZvttCte8jrf1KnfKSHbe0nwCXVxZC4EHk=
```

Comment:
Perhaps it can be removed by go mod tidy?

Reply:
It does not seem to want to budge. I did run `go mod tidy` before the PR commit, and tried again now, but no dice.